socorro-cmd

```diff
@@ -65,58 +84,47 @@ class Group:
 COMMANDS = [
     Group(
         'Crash processing utilities', {
-            'add_crashid_to_queue': 'socorro.scripts.add_crashid_to_queue.main',
-            'fetch_crashids': 'socorro.scripts.fetch_crashids.main',
-            'fetch_crash_data': 'socorro.scripts.fetch_crash_data.main',
-            'reprocess': 'socorro.scripts.reprocess.main',
+            'fetch_crashids': import_path('socorro.scripts.fetch_crashids.main'),
+            'fetch_crash_data': import_path('socorro.scripts.fetch_crash_data.main'),
+            'reprocess': import_path('socorro.scripts.reprocess.main'),
         }
     ),
     Group(
         'Local development environment maintenance', {
-            'create_recent_indices': (
+            'create_recent_indices': import_path(
                 'socorro.external.es.create_recent_indices_app.CreateRecentESIndicesApp'
             ),
             'clear_indices': 'socorro.external.es.clear_indices_app.ClearESIndicesApp',
-            'upload_telemetry_schema': (
+            'upload_telemetry_schema': import_path(
                 'socorro.external.boto.upload_telemetry_schema.UploadTelemetrySchema'
             ),
-            'db': 'socorro.scripts.db.main',
-            'pubsub': 'socorro.scripts.pubsub.main',
+            'db': import_path('socorro.scripts.db.main'),
+            'pubsub': import_path('socorro.scripts.pubsub.main'),
+            'depcheck': django_cmd('depcheck'),
```

Comment:
I added `depcheck` figuring I run it periodically. Today's me doesn't think we should add all the Django commands that we use, but we should add the ones that we use for general local dev environment management.

For example, I'm reworking all the "clean elasticsearch indices" which includes a couple of apps, a crontabber job, a script, a house in kentucky, an obsolete lotto ticket, and a horse with no name. Seems kind of silly to have all that, but we need the script which wipes Elasticsearch state for the local dev environment. I think that should be in `socorro-cmd`.

Anyhow, I added depcheck because I needed one to test with and I do periodically run this since it covers the whole codebase.